src/app/components/radio/index.html

```diff
@@ -1,16 +1,261 @@
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div class="sky-margin-stacked-content-section">
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-5.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use radio buttons for a long list of options.</div>
+                    </div>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-6.png" />
+                    <div class="caption caption-success sky-text-success">Do use different selection controls for more than 5 choices.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Don’t use radio buttons for two options that are exact opposites. Use a <a routerLink="/components/checkbox">checkbox</a> instead.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div class="sky-margin-stacked-content-section">
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-7.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use radio buttons to represent exact opposites.</div>
+                    </div>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-8.png" />
+                    <div class="caption caption-success sky-text-success">Do use a checkbox to represent exact opposites.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Don't use radio buttons when user selections significantly alter the workflow. Use <a routerLink="/components/action-button">action buttons</a> instead, and place them at the beginning of the task.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div class="sky-margin-stacked-content-section">
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-9.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use radio buttons for options that significantly alter the workflow.</div>
+                    </div>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-10.png" />
```

Comment:
The red line and caption under this image doesn't match the width of the image. I think it should be adjusted to be narrower.